sampledata/customspeech/en-US/training/structured-text.md

```diff
@@ -0,0 +1,19 @@
+@ list locale =
+- english
+- spanish
+- french
+- german
+
+@ list programming_language =
+- python
+- java
+- .net
+
+
+@ speech:phoneticlexicon
+- .net/D AO T N EH T
+
+# SomeTrainingSentence
+- Microsoft custom speech supports structured text input files in {@locale}

```

Comment:
I would suggest to capitalize Custom Speech, as we do in the docs (also next two lines)

Reply:
(assuming that case does not matter here)